Datastore/src/Query/Query.php

```diff
@@ -109,7 +115,10 @@ class Query implements QueryInterface
         '<=' => self::OP_LESS_THAN_OR_EQUAL,
         '>'  => self::OP_GREATER_THAN,
         '>=' => self::OP_GREATER_THAN_OR_EQUAL,
-        '='  => self::OP_EQUALS
+        '='  => self::OP_EQUALS,
+        '!='  => self::OP_NOT_EQUALS,
+        'IN'  => self::OP_IN,
```

Comment:
I don't think we need to add IN/NOT_IN as short operators, the concept here is to have a small translation layer for what the backend expects - but it seems like IN/NOT_IN should be accepted as is.

Reply:
Could we also please add a unit test for the not equals short operator? There should be some precedent already set to follow. :)